src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/ClientCredentialTypeTests.cs

```diff
@@ -28,13 +27,22 @@ public static void BasicAuthentication_RoundTrips_Echo()
         {
             BasicHttpBinding basicHttpBinding = new BasicHttpBinding(BasicHttpSecurityMode.Transport);
             basicHttpBinding.Security.Transport.ClientCredentialType = HttpClientCredentialType.Basic;
-            Action<ChannelFactory> credentials = (factory) =>
-            {
-                factory.Credentials.UserName.UserName = s_username;
-                factory.Credentials.UserName.Password = s_password;
-            };
 
-            ScenarioTestHelpers.RunBasicEchoTest(basicHttpBinding, Endpoints.Https_BasicAuth_Address, "BasicHttpBinding - Basic auth ", errorBuilder, credentials);
+            Logger.LogInformation("Starting basic echo test.\nTest variation:...\n{0}\nUsing address: '{1}'", "BasicAuthentication_RoundTrips_Echo", Endpoints.Https_BasicAuth_Address);
```

Comment:
Remove references to `Logger`, however do consider displaying this information if there's an exception thrown